back-end/app/Models/User.php

```diff
@@ -18,7 +18,8 @@ class User extends Authenticatable
      * @var array<int, string>
      */
     protected $fillable = [
-        'name',
+        'first_name',
+        'last_name',
         'email',
         'password',
```

Comment:
You can also use this one instead of fillable and list all possible fields which is kinda hassle when you have more than 10fields but I am not saying this is not good. It is good :D.
`protected $guarded = [];`